pkg/rootless/rootless_linux.c

```diff
@@ -457,6 +457,12 @@ create_pause_process (const char *pause_pid_file_path, char **argv)
             }
 
           r = TEMP_FAILURE_RETRY (write (p[1], "0", 1));
+          if (r < 0)
+            {
+              fprintf (stderr, "cannot write to pipe: %s\n", strerror (errno));
+              kill (pid, SIGKILL);
```

Comment:
if we kill the `pid` process we would also need to cleanup the pid file.  Although if we get this far, it is probably safer to just let it run and avoid to kill the pause process.  Could you drop the `kill (pid, SIGKILL)` here?

Reply:
Removed,  PTAL